src/utils/versioning.js

```diff
@@ -3,7 +3,7 @@ const SUPPORTED_VERSIONS = ['v0.31', 'v0.32'];
 /** latest version number for URLs without version prefix */
 const LATEST_VERSION = 'v0.33';
 
-const DEFAULT_JS_API_VERSIONS_TO_BUILD = 2;
+const DEFAULT_JS_API_VERSIONS_TO_BUILD = 10;
```

Comment:
@imiric I will rollback this change later. 

The goal is to improve building time by building only the latest two versions for the `development` and `staging` environments.  For `production`, it builds all the existing versions.

Reply:
Hmm OK, I'll remove it. I wasn't sure why I wasn't seeing the older versions locally, so I'll add a comment instead.